### PR TITLE
Update data changelog with LEI addition

### PR DESCRIPTION
**Pull request by StephenAbbott**

### CODE REVIEW

File: app/views/pages/data_changelog.html.haml [hunk 1]
```diff
@@ -21,9 +21,28 @@
...
+                Legal Entity Identifiers (LEIs) have been added as additional 
+                identifiers to enabling greater connectivity between international 
+                beneficial ownership  and corporate ownership datasets.
+              %p
+                This <a href="https://github.com/openownership/register-sources-oc/issues/8">work</a> 
+                has been enabled by an <a href="https://www.gleif.org/en/lei-data/lei-mapping/download-oc-to-lei-relationship-files">existing 
+                mapping process</a> between OpenCorporates IDs and LEIs.
+              %p
+                Technical details about mapping Open Ownership Register IDs to LEIs has been 
+                <a href="https://www.gleif.org/en/lei-data/lei-mapping/download-oc-to-lei-relationship-files/open-ownership-register-id-to-lei-relationship">
```
Comment (tiredpixel) [inline app/views/pages/data_changelog.html.haml:39]: This style of HTML-within-HAML will work, but a HAML-only alternative would be something like

```haml
%a{ href: 'https://www.gleif.org/en/lei-data/lei-mapping/download-oc-to-lei-relationship-files/open-ownership-register-id-to-lei-relationship' } published
```

etc.

File: app/views/pages/data_changelog.html.haml [hunk 2]
```diff
@@ -21,9 +21,28 @@
...
+                published</a> on the Global Legal Entity Identifier Foundation website.
+
+
+          %h2 22 July 2023
+          %p
+            <strong>First affected bulk download:</strong> 22 July 2023
```
Comment (tiredpixel) [inline app/views/pages/data_changelog.html.haml:45]: Or HAML-only could be something like:

```haml
%p
  %strong First affected […]
  22 July 2023
```

**Review by tiredpixel — DISMISSED** [review]
Optional improvements noted.

**Comment by tiredpixel** [comment]
In reference to https://github.com/openownership/register/issues/186 , I've merged master into this branch. That should activate the Heroku review apps for this PR.

**Review by tiredpixel — APPROVED** [review]

**Comment by tiredpixel** [comment]
@StephenAbbott, just a note that this has been approved, but not yet merged. Is there anything else you need from me?

**Comment by StephenAbbott** [comment]
Nope, @tiredpixel. I've merged now

**Pull request merged**